### PR TITLE
landing page updates

### PR DESCRIPTION
**Pull request by epwinchell**
Jira:  https://issues.redhat.com/browse/RHCLOUD-26464

- Section icon is a placeholder
- Landing content is visible on other pages

Screenshot 2023-06-22 at 1 21 13 PM
Screenshot 2023-06-22 at 1 20 00 PM

### CODE REVIEW

File: components/Navigation/SectionNavigation.tsx [hunk 1]
```diff
@@ -2,63 +2,83 @@ import Link from "next/link";
...
   CardBody,
   CardTitle,
+  Text,
+  TextContent,
+  TextVariants,
 } from "@patternfly/react-core";
 import { createUseStyles } from "react-jss";
+import classnames from "clsx";
+import CommandLineIcon from "./command-line";
+// import PenToolIcon from './pen-tool';
```
Comment (karelhala) [inline components/Navigation/SectionNavigation.tsx]: No need for commented out icons

File: components/Navigation/SectionNavigation.tsx [hunk 2]
```diff
@@ -2,63 +2,83 @@ import Link from "next/link";
...
+          >
+            <CardTitle className="pf-u-pb-md">
+              {title}
+              <Label
+                color="orange"
+                className={classnames(classes.label, "pf-u-float-right")}
+              >
+                <CommandLineIcon />
+              </Label>
+              {/*<Label color="purple" className={classnames(classes.label, "pf-u-float-right")}><PenToolIcon/></Label>*/}
```
Comment (karelhala) [inline components/Navigation/SectionNavigation.tsx:65]: Extra commented out label
```suggestion
```

File: components/Navigation/command-line.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+const CommandLineIcon = () => {
+  // eslint-disable-next-line max-len
+  return (
```
Comment (karelhala) [inline components/Navigation/command-line.tsx]: No need for return statement
```suggestion
  // eslint-disable-next-line max-len
const CommandLineIcon = () => (
```

File: components/Navigation/command-line.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            <path
+              d="M46.907489,0 L1.09251101,0 C0.489133842,0 0,0.489133842 0,1.09251101 L0,46.907489 C0,47.5108662 0.489133842,48 1.09251101,48 L46.907489,48 C47.1972409,48 47.4751254,47.8848965 47.6800109,47.6800109 C47.8848965,47.4751254 48,47.1972409 48,46.907489 L48,1.09251101 C48,0.802759077 47.8848965,0.524874626 47.6800109,0.319989067 C47.4751254,0.115103508 47.1972409,0 46.907489,0 Z M45.814978,45.814978 L2.18502203,45.814978 L2.18502203,2.18502203 L45.814978,2.18502203 L45.814978,45.814978 Z"
+              id="Shape"
+            ></path>
+          </g>
+        </g>
+      </g>
+    </svg>
+  );
+};
```
Comment (karelhala) [inline components/Navigation/command-line.tsx]:
```suggestion
);
```

File: components/Navigation/pen-tool.tsx [hunk 5]
```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const PenToolIcon = () => {
+  // eslint-disable-next-line max-len
+  return (
```
Comment (karelhala) [inline components/Navigation/pen-tool.tsx]: Same as above, no need for return statement
```suggestion
  // eslint-disable-next-line max-len
const PenToolIcon = () => (
```

File: components/Navigation/pen-tool.tsx [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+            <path d="M35.8153492,15.2812196 L35.8153492,9.97524191 C35.8056897,9.36677375 35.3096387,8.87859653 34.7010938,8.87867319 L29.3951162,8.87867319 C28.0509352,8.87867319 28.2985475,10.3997201 28.2985475,11.3194229 C22.010879,7.10965789 13.8044702,7.10965789 7.51680167,11.3194229 C7.51680167,10.3997201 7.74672737,8.89635979 6.40254637,8.87867319 L1.09656871,8.87867319 C0.490950536,8.87867319 0,9.36962373 0,9.97524191 L0,15.2812196 C0,15.8897644 0.488100557,16.3858154 1.09656871,16.3954749 L6.40254637,16.3954749 C7.6936676,16.3954749 7.51680167,14.9628609 7.51680167,14.0608447 C13.5716566,9.14422468 22.2436926,9.14422468 28.2985475,14.0608447 C28.2985475,14.9628609 28.103995,16.3601017 29.3951162,16.3954749 L34.7010938,16.3954749 C35.3164801,16.3954749 35.8153492,15.8966058 35.8153492,15.2812196 Z M5.30597765,14.1846508 L2.21082402,14.1846508 L2.21082402,11.0894972 L5.30597765,11.0894972 L5.30597765,14.1846508 Z M33.6045251,14.1846508 L30.5093715,14.1846508 L30.5093715,11.0894972 L33.6045251,11.0894972 L33.6045251,14.1846508 Z"></path>
+          </g>
+        </g>
+      </g>
+    </svg>
+  );
+};
```
Comment (karelhala) [inline components/Navigation/pen-tool.tsx]:
```suggestion
);
```

**Review by karelhala — APPROVED** [review]
Looking good! Just a few commented code, but overall looks good.

File: components/Navigation/SectionNavigation.tsx (hunk 2 above)
Comment (epwinchell) [reply components/Navigation/SectionNavigation.tsx:65]: @karelhala - we have two label variants and icons that need to be used in the cards (see mocks.) I included the second for your convenience.

Screenshot 2023-06-27 at 1 09 28 PM

File: components/Navigation/command-line.tsx (hunk 3 above)
Comment (epwinchell) [reply components/Navigation/command-line.tsx:5]: @karelhala This is how we've always done it and it's what I see in online examples. Can you point me to a better way?

**Review by karelhala — APPROVED** [review]

**Pull request merged**